test/parallel/test-whatwg-url-canparse.js

```diff
@@ -20,23 +20,29 @@ assert.throws(() => {
 assert.strictEqual(URL.canParse('https://example.org'), true);
 
 {
-  // V8 Fast API
-  function testFastPaths() {
-    // `canParse` binding has two overloads.
-    assert.strictEqual(URL.canParse('https://www.example.com/path/?query=param#hash'), true);
-    assert.strictEqual(URL.canParse('/', 'http://n'), true);
+  // Only javascript methods can be optimized through %OptimizeFunctionOnNextCall
+  // This is why we surround the C++ method we want to optimize with a JS function.
```

Comment:
I think is worthy to update the fast api docs to add this note, and maybe  the exception when it's called with C++, just to be easy to find the docs when someone receive this error